include/cross.h

```diff
@@ -57,6 +57,24 @@
 #define ftruncate(blah,blah2) chsize(blah,blah2)
 #endif
 
+/* Large file support */
+#if defined(_MSC_VER)
+	// MSVC doesn't support the posix fstream functions,
+	// typedef their equivalents
+	#define cross_ftello _ftelli64
+	#define cross_fseeko _fseeki64
+	#define cross_off_t __int64
+#else
+	// All other platforms should have POSIX fstream 'o' support
+	// Note: Meson automatically sets preprocessor defines for us
+
+	// Check that off_t is 64 bits
+	static_assert(sizeof(off_t) == sizeof(int64_t), "off_t not 64 bits");
+	#define cross_ftello ftello
+	#define cross_fseeko fseeko
+	#define cross_off_t off_t
+#endif
+
```

Comment:
In `_fat` and `_imgmount` there are three duplicate chunks of code that get the file size (and worse, they don't even check `fseek`'s return-value).

Suggest de-duping those three with an wrapper call here `cross_get_fsizeo`? (probably better names!), that leverages the wrappers (and can also check fseet's rcode).